pkg/scheduler/plugins/drf/drf.go

```diff
@@ -160,7 +160,7 @@ func (drf *drfPlugin) updateShare(attr *drfAttr) {
 
 func (drf *drfPlugin) calculateShare(allocated, totalResource *api.Resource) float64 {
 	res := float64(0)
-	for _, rn := range api.ResourceNames() {
+	for _, rn := range allocated.ResourceNames() {
```

Comment:
Might it be better to use `totalResource` because `allocated`'s `ScalarResources` perhaps is nil?

Reply:
Thanks for your review, updated